ansible_collections/arista/avd/molecule/eos_designs_unit_tests_v4.0/inventory/host_vars/evpn_services_l2_only_true.yml

```diff
@@ -7,5 +7,5 @@ l3leaf:
   nodes:
     - name: evpn_services_l2_only_true
       id: 101
-      bgp_as: 101
```

Comment:
1. Yes we want the 4.0 scenarios to match the data models of the schemas
2. Since we are disabling the convert_dicts in the v4.0 molecule scenarios some of the auto conversions do not work. (This one would though, but see pt.1 :)